src/bridge.ts

```diff
@@ -1183,12 +1235,15 @@ export class Bridge {
         let roomId = provisionedRoom.roomId;
         // If they didn't pass an existing `roomId` back,
         // we expect some `creationOpts` to create a new room
-        if (!roomId) {
-            // eslint-disable-next-line camelcase
-            const createRoomResponse: {room_id: string} = await this.botClient.createRoom(
+        if (roomId === undefined) {
+            roomId = await this.botIntent.botSdkIntent.underlyingClient.createRoom(
                 provisionedRoom.creationOpts
             );
-            roomId = createRoomResponse.room_id;
+        }
+
+        if (!roomId) {
+            // In theory this should never be called, but typescript isn't happy.
+            throw Error('Expected roomId to be defined');
```

Comment:
You already checked that it is defined.
```suggestion
            throw Error('Expected roomId to be truthy');
```

I guess `underlyingClient.createRoom()` can return something other than a string?

Reply:
It can't, but TS wasn't happy :/